Gui/opensim/CMakeLists.txt

```diff
@@ -16,6 +16,22 @@ endif()
 
 find_package(Ant REQUIRED)
 
+# http://xit0.org/2013/04/cmake-use-git-branch-and-commit-details-in-project/
+# Get the latest abbreviated commit hash of the working branch
+execute_process(
+  COMMAND git log -1 --format=%h
+  WORKING_DIRECTORY ${CMAKE_SOURCE_DIR}
+  OUTPUT_VARIABLE GIT_COMMIT_HASH
+  OUTPUT_STRIP_TRAILING_WHITESPACE
+)
+string(TIMESTAMP timestamp)
+set (BUILD_ID_RAW ${timestamp}.${GIT_COMMIT_HASH})
+# Replace illegal char in Path ":" with "."
+string(REPLACE ":" "." BUILD_ID ${BUILD_ID_RAW})
```

Comment:
Please consider using similar commands as I used in the CI scripts to change the version for the development builds.

https://github.com/opensim-org/opensim-gui/blob/3d029e5f6121d089da2dc4fd226e4607cb445e6c/.travis.yml#L44-L46

This way, the date comes from the git commit, not the arbitrary time when CMake is run.

Reply:
Done 👍